src/pybind/mgr/dashboard/frontend/src/app/shared/api/auth.service.ts

```diff
@@ -21,15 +21,23 @@ export class AuthService {
     return this.http.post('api/auth/check', { token: token });
   }
 
-  login(credentials: Credentials) {
+  getNeedChangePassword(username: string) {
+    return this.http
+      .post('api/auth/getNeedChangePassword', {username: username})
+      .toPromise()
+      .then((resp: any) => {
```

Comment:
As we have defined a type `LoginResponse`, why not enforcing that instead of using `any`?